src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -29,6 +29,24 @@ void HBProcessor::process(const cpp2::HBReq& req) {
         return;
     }
 
+    if (!clusterMan_) {
+        LOG(ERROR) << "Meta server init error!";
+        resp_.set_code(cpp2::ErrorCode::E_WRONGCLUSTER);
+        onFinished();
+        return;
+    }
+
+    ClusterID peerCluserId = req.get_clusterId();
+    if (peerCluserId == 0) {
+        LOG(INFO) << "Set clusterId for new host " << host << "!";
+        resp_.set_clusterId(clusterMan_->getClusterId());
+    } else if (peerCluserId != clusterMan_->getClusterId()) {
+        LOG(ERROR) << "Reject wrong cluster host " << host << "!";
+        resp_.set_code(cpp2::ErrorCode::E_WRONGCLUSTER);
```

Comment:
Thanks very much!

I have thought about this situation carefully! But I think now it's difficult to resolve this situation,because:
Possibly, meta server should store the identity info of metaclient(ie: ip:port) in case of metaclient sent an old clusterId and meta server don't reject the request of metaclient.

But, if clusterId loaded error, metaclient ip:port would loaded error probably(because clusterId and metaclient ip:port store in kvstore);